webpack.config.js

```diff
@@ -36,10 +38,24 @@ const vendorPaths = {
 // Cache-busting Jinja2 macros (`webpack-macros.html`) used in `layout.html`
 //
 function macroTemplate({ compilation }) {
+  // add a hash keep for each build
   const hash = compilation.hash;
+
   // We load these files into the theme via HTML templates
-  const css_files = ["styles/theme.css", "styles/pydata-sphinx-theme.css"];
-  const js_files = ["scripts/pydata-sphinx-theme.js"];
+  const css_files = [
+    "styles/theme.css",
```

Comment:
Could you add comments to each of these so it's clear what they are for (specifically what is the difference between `theme.css` and `pydata-sphinx-theme.css`?)